modules/installation/proc-installing-with-helm-cli.adoc

```diff
@@ -0,0 +1,56 @@
+[id="installing-with-helm-cli_{context}"]
+= Installing {product} with Helm CLI
+
+You can use the Helm CLI to install {product} on OpenShift or Kubernetes.
+
+.Prerequisites
+* An `oc` or `kubectl` session with a user that has the `developer` or `admin` role on an OpenShift or Kubernetes cluster.
+* Helm CLI.
+
+.Procedure
+. For a Kubernetes cluster, you  must first add the OpenShift Helm Charts repository. On Openshift, this is already present. 
++
+----
+$ helm repo add openshift-helm-charts https://charts.openshift.io/
+----
+
+. Create and activate the _<rhdh>_ OpenShift project or Kubernetes namespace:
++
+[subs="quotes+"]
+----
+$ NAMESPACE=_<rhdh>_
+$ oc new-project ${NAMESPACE} || oc project ${NAMESPACE}
+----
+
+. Install the {product-short} Helm Chart:
++
+[subs="attributes+"]
+----
+$ helm upgrade redhat-developer-hub -i https://github.com/openshift-helm-charts/charts/releases/download/redhat-redhat-developer-hub-{product-chart-version}/redhat-developer-hub-{product-chart-version}.tgz
+----
+
+. Collect values from your OpenShift or Kubernetes cluster:
++
+----
+$ PASSWORD=$(oc get secret redhat-developer-hub-postgresql -o jsonpath="{.data.password}" | base64 -d)
+$ CLUSTER_ROUTER_BASE=$(oc get route console -n openshift-console -o=jsonpath='{.spec.host}' | sed 's/^[^.]*\.//')
+----
+
+. Configure your {product-short} Helm Chart instance with the collected values:
++
+[subs="attributes+"]
+----
+$ helm upgrade redhat-developer-hub -i "https://github.com/openshift-helm-charts/charts/releases/download/redhat-redhat-developer-hub-{product-chart-version}/redhat-developer-hub-{product-chart-version}.tgz" \
+    --set global.clusterRouterBase="${CLUSTER_ROUTER_BASE}" \
+    --set global.postgresql.auth.password="$PASSWORD"
+----
+
+. Display the running {product-short} instance URL:
++
+----
+$ echo https://redhat-developer-hub-${NAMESPACE}.${CLUSTER_ROUTER_BASE}
+----
```

Comment:
```suggestion
+
[source,terminal]
----
echo https://redhat-developer-hub-<namespace>.<cluster_route_base>
----
```

Format all variables inline with how they're formatted throughout docs and according to https://redhat-documentation.github.io/supplementary-style-guide/#user-replaced-values